### ISSUE TITLE
ParsingException when multiple <code> blocks follow an annotation.

### ISSUE THREAD

**Issue by traceyyoshima** [openrewrite/rewrite#938]
```java
    @Test
    fun test(jp: JavaParser) = assertParsePrintAndProcess(
        jp, JavaTreeTest.NestingLevel.CompilationUnit, """
            /**
             * @return <code>true</code> if the given <code>source</code>.
             */
            public class A {
                void method() {}
            }
        """
    )
```

**Comment by traceyyoshima** [comment]
fixed by [dbda28b](https://github.com/openrewrite/rewrite/commit/dbda28bdf560f5f35572828e393b63fe6f86c168)

**Comment by traceyyoshima** [comment]
fixed by [dbda28b](https://github.com/openrewrite/rewrite/commit/dbda28bdf560f5f35572828e393b63fe6f86c168)
